### PR TITLE
Fix issue where user could get stuck in a loop of password prompts

### PR DESCRIPTION
**Pull request by ohayon**

### CODE REVIEW

File: credentials_manager/lib/credentials_manager/account_manager.rb [hunk 1]
```diff
@@ -13,7 +13,8 @@ def initialize(user: nil, password: nil, prefix: nil)
     end
 
     def user
-      @user ||= fetch_password_from_env
+      @user ||= ENV["FASTLANE_USER"]
```
Comment (KrauseFx) [inline credentials_manager/lib/credentials_manager/account_manager.rb]: Why?

Comment (ohwutup) [inline credentials_manager/lib/credentials_manager/account_manager.rb]: this is the user method. so we want to get the user from env vars, not the password. right?

Comment (ohwutup) [inline credentials_manager/lib/credentials_manager/account_manager.rb]: a failing test surfaced this

Comment (KrauseFx) [inline credentials_manager/lib/credentials_manager/account_manager.rb]: Whooops, sorry about that

Comment (KrauseFx) [inline credentials_manager/lib/credentials_manager/account_manager.rb]: So good that we had tests 👍

**Comment by KrauseFx** [comment]
Looks great 🚀

**Pull request merged**